README.md

```diff
@@ -13,44 +13,80 @@ It includes [JSON Web Token (JWT)](https://oauth.net/2/jwt/) verification, and [
+   1. Select the **Web application** type and give your app a name.
+   1. Add `http://127.0.0.1:7676` and `http://127.0.0.1:7676/callback` to **Authorized JavaScript origins** and **Authorized redirect URIs**, respectively. This is for local testing only; remember to remove these URLs later!
+   1. Tap **Create**.
+1. Store your newly created credentials in new (gitignored) files in the root of your Compute project:
+   - Paste the *client ID* in `.secret.client_id`
+   - Paste the *client secret* in `.secret.client_secret` 
+   - Type a long, non-guessable random string of your choice into `.secret.nonce_secret`
+      ```sh
+      dd if=/dev/random bs=32 count=1 | base64 > .secret.nonce_secret
+      ```
+1. Fetch Google's OIDC Discovery document from [accounts.google.com/.well-known/openid-configuration](https://accounts.google.com/.well-known/openid-configuration). Paste the JSON-stringified contents of the file into the `openid_configuration` property in `fastly.toml`, under `[local_server.config_stores.oauth_config.contents]`.
+   ```sh
+   curl -s https://accounts.google.com/.well-known/openid-configuration | jq -c @json
+   ```
+1. Note the `jwks_uri` property inside the OIDC Discovery document. Fetch the document at that URL and paste its JSON-stringified contents into the `jwks` property in `fastly.toml`, under `[local_server.config_stores.oauth_config.contents]`.
+   ```sh
+   curl -s https://www.googleapis.com/oauth2/v3/certs | jq -c @json
+   ```
+
+### 2. Test your configuration locally
+
+Spin up the local development server for your Compute service:
 
-As an example, if you are using Auth0, follow these steps after installing the starter kit:
+```term
+fastly compute serve
+```
 
-1. In the [Auth0 dashboard](https://manage.auth0.com/), choose **Create Application**. Give your app a name and choose "Regular web application".
-   - The *client ID* (eg. `4PWZBMqMWxnKXt1heitack0Jy2xRQP0p`) is shown next to your application name.
-1. Open `src/config.toml` in your Fastly project and paste in the `client_id` from your IdP.  Set the `nonce_secret` field to a long, non-guessable random string of your choice.  Save the file.
-1. Back in Auth0's dashboard, click **Settings**, and note down the *authorization server* hostname (eg. `dev-wna8lqtb.us.auth0.com`) is shown in the **Domain** field.
-1. In a new tab, navigate to `https://{authorization-server-hostname}/.well-known/openid-configuration`.  Save it to `src/well-known/openid-configuration.json` in your Fastly project.
-1. Open the file you just created and locate the `jwks_uri` property.  Fetch the document at that URL and save it to `src/well-known/jwks.json` in your Fastly project.
+Browse to http://127.0.0.1:7676. If everything is configured correctly, you should be able to complete an end-to-end OAuth 2.0 flow.
 
-### Deploy the Fastly service and get a domain
+### 3. Deploy the Fastly service and get a domain
 
```

Comment:
You need to mention that the `COOKIE_PREFIX` needs to be changed between local and production.  Or, modify the code that uses the cookie prefix to automatically use the right prefix, see my comment in `cookies.rs`.

Reply:
I've modified the code so that it automatically uses the right prefix :)